src/apis/user/user.resolver.ts

```diff
@@ -17,13 +17,26 @@ export class UserResolver {
 
   //닉네임 체크
   @Mutation(() => String)
-  async checkNickName(
+  async checkNickname(
     @Args('nickname') nickname: string, //
   ) {
-    const findNickName = await this.userService.findOne({ nickname });
-    if (findNickName) {
+    const findNickname = await this.userService.findOne({ nickname });
+    if (findNickname) {
       return false;
-    } else return true;
+    }
+    return true;
+  }
+
+  //이메일 체크
+  @Mutation(() => String)
+  async checkEmail(
+    @Args('email') email: string, //
+  ) {
+    const findEmail = await this.userService.findOne({ email });
```

Comment:
```suggestion
    const user = await this.userService.findOne({ email });
```
해당 변수는 user 객체를 가지고있을거같은데 findEmail 은 마치 함수명처럼 동사로 시작하기때문에 적절하지 않은 변수명인듯하네요

Reply:
반영하겠습니다!